test/Microsoft.Azure.Devices.Edge.Test/PriorityQueues.cs

```diff
@@ -187,5 +266,19 @@ private async Task<PriorityQueueTestStatus> PollUntilFinishedAsync(string module
             while (!testStatus.IsFinished);
             return testStatus;
         }
+
+        private TestInfo InitTestInfo(int length, int ttlThreshold, bool defaultTtls = false)
+        {
+            List<int> ttls = defaultTtls ? this.BuildDefaultTtls(length) : this.BuildTtls(length, ttlThreshold);
+            List<int> priorities = this.BuildPriorities(length);
+            return new TestInfo() { Ttls = ttls, Priorities = priorities, TtlThreshold = ttlThreshold };
```

Comment:
if defaultTtls is true and ttlThreshold is not used in line 272, why return ttlThreshold here?

Reply:
It's used in BuildAddInitialConfig. I didn't have it there at the time of you reviewing, but it should have been. Added it now.